### PR TITLE
US-1889327: Header fixes

### PR DESCRIPTION
**Pull request by poojapoojari**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by poojapoojari** [comment]
Hello @adegeo!

The 6th PR with 20 file edits is ready for technical review. Could you please review this PR?
Thank you!

File: dotnet-desktop-guide/framework/wpf/controls/how-to-create-apply-template.md [hunk 1]
```diff
@@ -35,11 +36,9 @@ On the other hand, if you need a control with new functionality, different prope
...
 Create a new WPF application and in *MainWindow.xaml* (or another window of your choice) set the following properties on the **\<Window>** element:
 
-|     |     |
-| --- | --- |
-| **Title**         | `Template Intro Sample` |
-| **SizeToContent** | `WidthAndHeight` |
-| **MinWidth**      | `250` |
+| Title | SizeToContent | MinWidth |
+|-------|---------------|----------|
+| `Template Intro Sample` | `WidthAndHeight` | `250` |
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/how-to-create-apply-template.md:41]: This is better the other way around. You can use Property / Value for the headers
```suggestion
| Property          | Value                   |
| ----------------- | ----------------------- |
| **Title**         | `Template Intro Sample` |
| **SizeToContent** | `WidthAndHeight`        |
| **MinWidth**      | `250`                   |
```

File: dotnet-desktop-guide/framework/wpf/controls/how-to-create-apply-template.md [hunk 2]
```diff
@@ -63,10 +62,9 @@ To start with, add a `Window.Resources` element to your *MainWindow.xaml* file:
 
 Create a new **\<ControlTemplate>** with the following properties set:
 
-|     |     |
-| --- | --- |
-| **x:Key**         | `roundbutton` |
-| **TargetType**    | `Button` |
+| x:Key | TargetType |
+| ----- | ---------- |
+| `roundbutton` | `Button` |
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/how-to-create-apply-template.md:67]: Property/Value as headers here too

File: dotnet-desktop-guide/framework/wpf/controls/how-to-retrieve-a-text-selection.md [hunk 3]
```diff
@@ -13,14 +15,14 @@ ms.assetid: d5793172-1e11-4a39-9be0-73f336ed858d
...
-## Example  
+## Define a TextBox control
  The following [!INCLUDE[TLA#tla_xaml](../../../includes/tlasharptla-xaml-md.md)] example shows the definition of a <xref:System.Windows.Controls.TextBox> control that contains some text to select, and a <xref:System.Windows.Controls.Button> control with a specified <xref:System.Windows.Controls.Button.OnClick%2A> method.  
   
  In this example, a button with an associated <xref:System.Windows.Controls.Primitives.ButtonBase.Click> event handler is used to retrieve the text selection. When the user clicks the button, the <xref:System.Windows.Controls.Button.OnClick%2A> method copies any selected text in the textbox into a string. The particular circumstances by which the text selection is retrieved (clicking a button), as well as the action taken with that selection (copying the text selection to a string), can easily be modified to accommodate a wide variety of scenarios.  
   
  [!code-xaml[TextBox_MiscCode#_TextBoxSelectTextXAML](~/samples/snippets/csharp/VS_Snippets_Wpf/TextBox_MiscCode/CSharp/Window1.xaml#_textboxselecttextxaml)]  
   
-## Example  
+## C# example on OnClick event handler
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/how-to-retrieve-a-text-selection.md:25]: This also has VB.NET code so no reason to point out C# here
```suggestion
## OnClick event handler
```

**Review by adegeo — CHANGES_REQUESTED** [review]
Thanks! some minor things to change.

**Comment by poojapoojari** [comment]
Hello @adegeo,

Thank you for your feedback! I have updated the content for the same. Could you please have a look and approve the changes?

Thank you!

**Review by adegeo — APPROVED** [review]
Looks good, thanks!

**Pull request merged**